### ISSUE TITLE
General error: 1 too many SQL variables

### ISSUE THREAD

**Issue by devfaysal** [calebporzio/sushi#56]
Thanks for fixing #38 
This problem is not directly relevant to Sushi, but want to know if there are any limitations of number of raws in Sushi.

Please check Travis build error: https://travis-ci.org/devfaysal/laravel-bangladesh-geocode/jobs/660522546?utm_medium=notification&utm_source=github_status

Below are the details of the issue:
I have a package [devfaysal/laravel-bangladesh-geocode](https://github.com/devfaysal/laravel-bangladesh-geocode) which provide all of Division, District and Upazila data for form dropdown. I was using laravel migrations and Seeder.

Now I am rewriting that package using Sushi so that we do not need any migration and seeder.
In my local setup (Ubuntu 18.04, PHP 7.4) everything is fine, all tests passed. but in Travis, tests are failing for php 7.2 and 7.3 with the error `General error: 1 too many SQL variables`, but passing for 7.4

I searched in google for the solution, got some reply that SQLite has limitation of 999 parameters.
But I wonder how it is passing for 7.4

**Comment by K2ouMais** [comment]
Yes, it seems Sushi comes only in question when you have smaller list.

I was getting the same error when trying to import 2000 rows.

SQLite has a limitation of 999 for this type of thing.

**Comment by muhsenmaqsudi** [comment]
@calebporzio please merge this pull request and update the package, since this is a big issue on our servers...

**Comment by K2ouMais** [comment]
@muhsenmaqsudi 
I dont think it will get merged, because all tests failed.
